gobblin-service/src/main/java/org/apache/gobblin/service/modules/flowgraph/Dag.java

```diff
@@ -48,6 +48,8 @@ public class Dag<T> {
   // Map to maintain parent to children mapping.
   private Map<DagNode, List<DagNode<T>>> parentChildMap;
   private List<DagNode<T>> nodes;
+  @Setter
```

Comment:
because we do not persist dag level field in mysql, adding fields to Dag will not be much useful  and may lead to bugs

Reply:
We want to have this field here as we didn't want to add additional parameters in all the methods to pass on is_failed value.